.github/workflows/docker-publish.yml

```
@@ -0,0 +1,49 @@
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Extract version number from release tag
+        run: |
+          TAG_NAME=${GITHUB_REF#refs/tags/}
+          DOJO_VERSION=${TAG_NAME#v}
+          echo "DOJO_VERSION=$DOJO_VERSION" >> $GITHUB_ENV
+
+      - name: Build and push docker image (preview)
+        if: ${{ contains(needs.prepare.outputs.tag_name, 'preview') }}
```

actionlint (1.7.4)

property "prepare" is not defined in object type {}

(expression)